docker_dev.sh

```diff
@@ -3,7 +3,7 @@ docker build -f alpine.docker -t diginc/pi-hole:alpine .
 docker tag diginc/pi-hole:alpine diginc/pi-hole:latest
 docker build -f debian.docker -t diginc/pi-hole:debian .
 
-IP=$(ip addr show eth0 | grep "inet\b" | awk '{print $2}' | cut -d/ -f1)
+IP=$(ip route get 8.8.8.8 | awk '{ print $NF; exit }')
```

Comment:
I'd posit that having a vpn tunnel is the exceptional configuration. 

Also, on Ubuntu 16.04+ network interfaces are 'consistent' ie the are NOT eth0.